### ISSUE TITLE
Make use of getty format description links from `format` field, link out to some descriptions `Newspapers | http://vocab.getty.edu/aat/300026656"`

### ISSUE THREAD

**Issue by sourslaw** [UMNLibraries/cdm-blacklightify#123]
Info button/icon or mouseover text to display more info on the format, e.g. _A cyanotype is....._
Getty provides "offical definition"
